### ISSUE TITLE
Request:  Discord Messages

### ISSUE THREAD

**Issue by Michaelawilliams28** [Joe-Klauza/sandstorm-admin-wrapper#56]
Is there any plans in the future for Discord messages? Server sends automated messages to Discord channel.
I have seen really helpful messages that I like to use with my current servers.
- Server stop and start (This one I know I can use a script to send alert when the services status changes)
- Play Join Messages (I have only seen this one once, I don't think I can get a script to query server)
- Server Backup messages (I probably can make a script with send alert after tar ball)

I am not the best with scripting but I usually can scrape something together.

**Comment by Joe-Klauza** [comment]
Definitely! I have first-hand experience implementing this in Ruby, so I'll add it to my to-do list. Likely features:
- Messages when the server goes down and comes back up
- "Live" player list and feed (joined/left) in the desired channel; the bot updates the same messages at the desired interval, so lock down the channel and just have the two messages. Player list message automatically splits when it hits Discord's message limit.
